authLdap.php

```diff
@@ -565,9 +563,9 @@ function wp_setcookie($username, $password, $already_md5 = false, $home = '', $s
  * he isn't
  * @conf boolean authLDAP
  */
-function authLdap_show_password_fields($return, $user)
+function authLdap_show_password_fields($user)
```

Comment:
As this is a callback for the filter ```show_password_fields``` the parameters need to stay like they where. (https://developer.wordpress.org/reference/hooks/show_password_fields/)